src/docs/product/cli/dif.mdx

```diff
@@ -272,6 +272,8 @@ sentry-cli upload-proguard \
     app/build/outputs/mapping/release/mapping.txt
 ```
 
+After the upload, Sentry deobfuscates future events. If you want to send an obfuscated crash to Sentry to verify correct operation, ensure that the ProGuard mapping files are listed in Project Settings > ProGuard.
```

Comment:
```suggestion
After the upload, Sentry deobfuscates future events. If you want to send an obfuscated crash to Sentry to verify the correct operation, ensure that the ProGuard mapping files are listed in Project Settings > ProGuard.
```

Reply:
Should we maybe mention the previous location for enterprise customers?

Reply:
don't think so, the sentence above is about debug files, it'd be natural to just read the sentence below and get it.